be/Cf.Domain/Enums/CarProblemType.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Cf.Domain.Enums;
+
+public enum ProblemType
```

Comment:
I would name the class and the file the same name. I would suggest to change file name to `ProblemType`